common/pb/serverinfo_chat_message.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
+/*
+ * Chat communication of a user to a target.
+ * Targets can be users or rooms.
+ * These communications are also stored in the DB log table.
+ */
+message ServerInfo_ChatMessage {
+    optional string time = 1; // time chat was sent
+    optional string sender_id = 2; // id of sender
+    optional string sender_name = 3; // name of sender
+    optional string sender_ip = 4; // ip of sender
```

Comment:
The protocol buffer message is designed around what data is stored in the chat log database table.  The message was created primarily for using the information when working with the log data stored in the db and the thought was for when we work with querying the data from the DB table for historic log lookup (mod only use).  The room history functionality uses the message container as a method to store the fields its needs into a qlist stored in memory but only assigns data to a couple of the message fields.  The qlist is then returned to the client by a room say message on join.